<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="titleFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="minimumSize">
         <size>
          <width>15</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="itemLayout">
      <item>
       <layout class="QHBoxLayout" name="renderLayerLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Env Layer: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="singleFrameButton">
          <property name="toolTip">
           <string>Configure single frame for rendering for the Env layer</string>
          </property>
          <property name="text">
           <string>Single Frame</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="overrideButton">
          <property name="toolTip">
           <string>Override the frame range for Env layer</string>
          </property>
          <property name="text">
           <string>Override</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="startFrameBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Start frame to override</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>End</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="endFrameBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>End frame to override</string>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Render Layers: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>singleFrameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>startFrameBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleFrameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>endFrameBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleFrameButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>overrideButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>startFrameBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overrideButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>endFrameBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
